### PR TITLE
Windows Static Analyzer

### PR DESCRIPTION
**Pull request by ajinabraham**
- New Branch for Testing @DominikSchlecht 's Code
### What has been implemented
- Static analysis of windows phone apps (currently only appx)
  - Parsing the xml-files
  - Static analysis with binskim and binscope
- Sadly a windows vm is needed, but a setup-script is located in install/windows/setup.py and the install/windows/readme.md gives a step by step tutorial
- Tested with 4 different apps; Setup tested on a blank windows 10 64x
### What has to be done before pushing
- Correct download-paths in install/windows/setup.py, install/windows/readme.md and install/windows/config.txt (I left the download path to my repository so you could test the code without pushing the files to your repo)
### What's to come
- Support for other windows phone file formats
- Deeper analysis
- Moving the windows vm to a docker image (don't know yet if this will work out and there are quite some showstoppers right now)

If you have questions or if I should rework some files, please feel free to contact me here or via e-mail.

Greetings,
Dominik

### CODE REVIEW

File: install/windows/config.txt [hunk 1]
```diff
@@ -11,15 +11,15 @@ url = https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
...
-[autostart]
-file = mobsf_rpc.bat
+nuget = Microsoft.CodeAnalysis.BinSkim
+
+[binscope]
+url = https://www.microsoft.com/en-us/download/details.aspx?id=44995
+file = C:\MobSF\Tools\BinScope\Binscope.exe
+
+[rpc]
+url = https://raw.githubusercontent.com/ajinabraham/Mobile-Security-Framework-MobSF/windows_static/install/windows/rpc_client.py
```
Comment (ajinabraham) [inline install/windows/config.txt]: Needs to be changed when merging to master

File: install/windows/setup.py [hunk 2]
```diff
@@ -16,8 +16,8 @@
 
 # Only static URL, let's hope this never changes..
 CONFIG_URL = (
-    "https://raw.githubusercontent.com/DominikSchlecht/"
-    "Mobile-Security-Framework-MobSF/master/install/windows/config.txt"
+    "https://raw.githubusercontent.com/ajinabraham/"
+    "Mobile-Security-Framework-MobSF/windows_static/install/windows/config.txt"
```
Comment (ajinabraham) [inline install/windows/setup.py]: Needs to be changed when merging to master

**Comment by ajinabraham** [comment]
Test Completion Status

- [x] Test the VM Approach with Windows 7 64bit SP1 and Windows 10 VMs.
- [ ] Test on Windows 7 and Windows 10/8 Host

**Comment by DominikSchlecht** [comment]
Merging should be good again 👍 :)

**Comment by ajinabraham** [comment]
@DominikSchlecht 

I wanted to test Windows App Static Analysis when host OS is Windows. Do you have instructions on how to configure when performing static analysis in Host Windows 7?

**Comment by DominikSchlecht** [comment]
@ajinabraham just run MobSF as you would normally do. On the first run the automatic installer should kick in and everything should work magically :)

**Comment by ajinabraham** [comment]
Awesome. Will test it and keep you posted

**Comment by ajinabraham** [comment]
Merging to master

**Pull request merged**